_studio/mfx_lib/vpp/src/mfx_vpp_utils.cpp

```diff
@@ -1337,6 +1337,14 @@ mfxStatus CheckFrameInfo(mfxFrameInfo* info, mfxU32 request, eMFXHWType platform
         case MFX_FOURCC_Y410:
             MFX_CHECK(platform >= MFX_HW_ICL, MFX_ERR_INVALID_VIDEO_PARAM);
             break;
+#endif
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+        case MFX_FOURCC_P016:
+        case MFX_FOURCC_Y216:
+        case MFX_FOURCC_Y416:
+            if (platform < MFX_HW_TGL_LP)
+                return MFX_ERR_INVALID_VIDEO_PARAM;
```

Comment:
Does libva has corresponding query? If not what is libva bug number to request it?